cmake/modules/DarwinSDKs.cmake

```diff
@@ -4,7 +4,7 @@ option(SWIFT_ENABLE_IOS32
 
 if(SWIFT_ENABLE_IOS32)
 set(SUPPORTED_IOS_ARCHS "armv7;armv7s;arm64;arm64e")
-set(SUPPORTED_IOS_SIMULATOR_ARCHS "i386;x86_64;arm64")
```

Comment:
(optional) This could be moved out of the if statement

Reply:
I will update it, thanks!